tsconfig.json

```diff
@@ -2,6 +2,8 @@
 // IDEs and TSLint. For IDEs it ensures that `experimentalDecorator` warnings are not showing up.
 {
   "compilerOptions": {
+    // Needed for Moment.js since it doesn't have a default export.
+    "allowSyntheticDefaultImports": true,
```

Comment:
I don't think we need that anymore right?

Reply:
We need it if we want webstorm to not complain about `import {default as moment} from 'moment'`, not sure if we care about that or not...